README.md

```diff
@@ -4,6 +4,28 @@ Web App and Servers for the MoU Dashboard
 A front-end to Smartsheet docs which allows PIs to easily report to the ICC their
 Statements of Work in accordance with MoUs.
 
+
+## REST Server
+A REST server that interfaces with a local MongoDB server *(future: also Smartsheet)*
+
+### Getting Started
+    python3 -m virtualenv -p python3 mou-dash-rest
+    . mou-dash-rest/bin/activate
+    pip install -r rest_server/requirements.txt
+
+#### *Optional:* First Kill All Active MongoDB Daemons
+`sudo killall mongod`
```

Comment:
This may not always work, if mongod is running as a service and gets restarted automatically.

Reply:
Do you recommend I just scrap these lines?

Reply:
Maybe just keep a note to kill existing mongods without saying how?

Reply:
3da7df1